js/components/analysisExecution/analysis-execution-list.js

```diff
@@ -152,7 +152,7 @@ define([
     startPolling() {
       this.pollId = this.PollService.add({
         callback: silently => this.loadData({ silently }),
-        interval: 10000,
+        interval: config.pollInterval,
```

Comment:
Default polling interval is 1 minute. It was 10 seconds, so maybe add different polling intervals to the settings to keep the user experience?

Reply:
I think we should use consistent poll intervals across the polling....for now, we should fix the bug where people used hard-coded poll intervals, but later we can introduce new configuration params for different poll contexts if we need it.